plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryConnectorTest.java

```diff
@@ -13,6 +13,7 @@
  */
```

Comment:
I am a bit lost. Are we testing bigquery here? @ebyhr can you please let me know what is the point of test based on incorrect column names?

As I understand we should only update `filterColumnNameTestData`

Reply:
Testing unsupported column names helps us find SQL injection vector.